RedHatInsights/insights-inventory-frontend
# fix(ESSNTL-4264): Remove link from inventory groups empty state

Fixes https://issues.redhat.com/browse/ESSNTL-4264.

Remove obsolete links from inventory groups empty states.

## How to test

1. When you don't have any groups created, make sure the empty state doesn't render any secondary links,
2. Check the same for the group without any systems.

## Screenshots

![image](https://github.com/RedHatInsights/insights-inventory-frontend/assets/31385370/453fc69e-ffe2-4f50-818c-5085d8c13929)
